### ISSUE TITLE
Trace mode can forget about past processes

### ISSUE THREAD

**Issue by vemv** [raboof/nethogs#65]
Hi,

I found that trace mode, against what I'd have expected, 'forgets' about past processes/programs that become inactive. Real production output:

`nethogs -t eth0 -d 300 -v 3`

```
Refreshing:
java/868/1000     4.4044  1.32513
chromium-browser/6455/1000        0.0257711       0.0216389
unknown TCP/0/0   0       0

Refreshing:
java/868/1000     4.5485  1.36938
unknown TCP/0/0   0       0
```

In the second 'round', chromium-browser disappeared, because consumption was too low. I don't think that makes sense for a process which runs forever, and sums results (note the `-v 3` flag).

Imagine that 1 hour later, chromium-browser becomes active again. Then would the past chromium-browser results be forgotten (excluded from the sum)? That'd be quite buggy imo!

Cheers - Victor

**Comment by raboof** [comment]
Yes - this problem was reported as #39 and I have a fix lined up in #56 - I'll try and merge that soon, in the mean time you're quite welcome to test it :)

**Comment by vemv** [comment]
Hey there,

I'll be happy to try it out in production if you tell me how to uninstall my current version first! :) I think it'd be a good thing to document in the README.

**Comment by raboof** [comment]
`sudo make install` will simply overwrite the old version. If you want to remove Nethogs from your system, you can use `sudo make uninstall`
